### PR TITLE
Update index.md - Increase readability

### PR DESCRIPTION
**Pull request by lukemurraynz**
Increase readability

### CODE REVIEW

**Comment by stephen-sumner** [comment]
@lukemurraynz - Many great suggestions! Thank you! I'll work on addressing your comments.

**Comment by ShannonLeavitt** [comment]
IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by BryanLa** [comment]
#label:"cact-review-in-progress"

File: docs/ready/landing-zone/index.md [hunk 1]
```diff
@@ -16,40 +16,40 @@ An Azure landing zone is an environment that follows key design principles acros
...
-| Shared management | With technology platforms such as AKS or AVS, a central IT team manages the underlying service. The application teams are responsible for the applications running on top of the technology platforms. You need to use different controls or access permissions for this model. These controls and permissions differ from the ones you use to centrally manage application landing zones.  
+| Central team management | A central IT team fully operates the landing zone. The team applies controls and platform tools to the platform and application landing zones.
+| Application team management | A platform administration team delegates the entire application landing zone to an application team. The application team manages and supports the environment. The management group policies ensure the platform team still governs the application landing zone. You can add other policies at the subscription scope and use alternative tooling for deploying, securing, or monitoring application landing zones.|
```
Comment (BryanLa) [inline docs/ready/landing-zone/index.md:43]:
```suggestion
| Application team management | A platform administration team delegates the entire application landing zone to an application team. The application team manages and supports the environment. The management group policies ensure that the platform team still governs the application landing zone. You can add other policies at the subscription scope and use alternative tooling for deploying, securing, or monitoring application landing zones.|
```

File: docs/ready/landing-zone/index.md [hunk 2]
```diff
@@ -16,40 +16,40 @@ An Azure landing zone is an environment that follows key design principles acros
...
 
-**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
+**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by the management group. The "Platform" management group subscriptions represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```
Comment (BryanLa) [inline docs/ready/landing-zone/index.md:28]:
```suggestion
**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by the management group. The subscriptions under the "Platform" management group  represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```

File: docs/ready/landing-zone/index.md [hunk 3]
```diff
@@ -16,40 +16,40 @@ An Azure landing zone is an environment that follows key design principles acros
...
 
-**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
+**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by the management group. The subscriptions under the "Platform" management group  represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```
Comment (BryanLa) [inline docs/ready/landing-zone/index.md:28]:
```suggestion
**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group  represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```

**Comment by BryanLa** [comment]
#label:"cact-review-in-progress"

File: docs/ready/landing-zone/index.md [hunk 4]
```diff
@@ -16,40 +16,40 @@ An Azure landing zone is an environment that follows key design principles acros
...
 
-**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
+**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group  represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```
Comment (BryanLa) [inline docs/ready/landing-zone/index.md:28]:
```suggestion
**Resource organization:** The conceptual architecture shows a sample management group hierarchy. It organizes subscriptions (yellow boxes) by management group. The subscriptions under the "Platform" management group represent the platform landing zones. The subscriptions under the "Landing zone" management group represent the application landing zones. The conceptual architecture shows five subscriptions in detail. You can see the resources in each subscription and the policies applied.
```

File: docs/ready/landing-zone/index.md [hunk 5]
```diff
@@ -33,23 +33,23 @@ An Azure landing zone consists of platform landing zones and application landing
...
+Accelerators are infrastructure-as-code implementations that help you deploy an Azure landing zone correctly. We have a platform landing zone accelerator and several application landing zone accelerators you can deploy.
 
 ### Platform landing zone accelerator
 
-There's a ready-made deployment experience called the **Azure landing zone portal accelerator**. The Azure landing zone portal accelerator deploys the conceptual architecture (*see figure 1*) and applies predetermined configurations to key components such as management groups and policies. It's suitable for organizations where the conceptual architecture aligns with the planned operating model and resource structure.
+There's a ready-made deployment experience called the **Azure landing zone portal accelerator**. The Azure landing zone portal accelerator deploys the conceptual architecture (*see figure 1*) and applies predetermined configurations such as management groups and policies. It suits organizations whose conceptual architecture aligns with the planned operating model and resource structure.
```
Comment (BryanLa) [inline docs/ready/landing-zone/index.md:52]:
```suggestion
There's a ready-made deployment experience called the **Azure landing zone portal accelerator**. The Azure landing zone portal accelerator deploys the conceptual architecture (*see figure 1*) and applies predetermined configurations to key components such as management groups and policies. It suits organizations whose conceptual architecture aligns with the planned operating model and resource structure.
```

**Comment by BryanLa** [comment]
@lukemurraynz - thanks for your contribution. I had to revert a few of the changes for style guide alignment and technical correctness. 

CACT #sign-off
#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Pull request merged**